<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleSIVPage2</class>
 <widget class="QWidget" name="VehicleSIVPage2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QLabel
{
	font: 20px, &quot;微软雅黑&quot;;
	color:white;
background-color:blue;
}
QPushButton
{
	background-color:brown ;
	border-top: 2px solid white;	
	border-left: 2px solid white;
}</string>
  </property>
  <widget class="QLabel" name="BCTTC2lbl">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>320</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCTTC1lbl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCTM1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCChargeCurrentTC1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCTM2lbl">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>320</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_12">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>335</y>
     <width>681</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_19">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>95</y>
     <width>681</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>蓄电池充电机
电流(A)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCChargeCurrentTC2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCChargeCurrentM1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>蓄电池温度(℃)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCChargeCurrentM2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_21">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>蓄电池母线
电压(V)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCVoltageTC1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_22">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>175</y>
     <width>681</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCVoltageM2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCVoltageTC2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCVoltageM1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_23">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>蓄电池母线
电流(A)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCCurrentTC1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_24">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>255</y>
     <width>681</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCCurrentM2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCCurrentTC2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBCCCurrentM1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_13">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>250</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:gray;
border-radius:10px;
font:14px;
color:yellow;
background-color:darkgray;</string>
   </property>
   <property name="text">
    <string>






第2页/共2页</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="NextPageBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>260</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font:30px;
color:black;
background-color:orange;
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>︿</string>
   </property>
  </widget>
  <zorder>lbl1_22</zorder>
  <zorder>lbl1_12</zorder>
  <zorder>lbl1_19</zorder>
  <zorder>lbl1_24</zorder>
  <zorder>BCTTC2lbl</zorder>
  <zorder>BCTTC1lbl</zorder>
  <zorder>BCTM1lbl</zorder>
  <zorder>LBLBCCChargeCurrentTC1</zorder>
  <zorder>BCTM2lbl</zorder>
  <zorder>lbl1_20</zorder>
  <zorder>LBLBCCChargeCurrentTC2</zorder>
  <zorder>LBLBCCChargeCurrentM1</zorder>
  <zorder>lbl1_6</zorder>
  <zorder>LBLBCCChargeCurrentM2</zorder>
  <zorder>lbl1_21</zorder>
  <zorder>LBLBCCVoltageTC1</zorder>
  <zorder>LBLBCCVoltageM2</zorder>
  <zorder>LBLBCCVoltageTC2</zorder>
  <zorder>LBLBCCVoltageM1</zorder>
  <zorder>lbl1_23</zorder>
  <zorder>LBLBCCCurrentTC1</zorder>
  <zorder>LBLBCCCurrentM2</zorder>
  <zorder>LBLBCCCurrentTC2</zorder>
  <zorder>LBLBCCCurrentM1</zorder>
  <zorder>lbl1_13</zorder>
  <zorder>NextPageBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
